ols/app/models/config.py

```diff
@@ -577,13 +577,23 @@ class SseTransportConfig(BaseModel):
     headers: dict[str, str] = Field(default_factory=dict)
 
 
+class StreamableHttpTransportConfig(BaseModel):
+    """Streamable HTTP transport configuration for MCP server."""
+
+    url: str
+    timeout: int = constants.STREAMABLE_HTTP_TRANSPORT_DEFAULT_TIMEOUT
+    sse_read_timeout: int = constants.STREAMABLE_HTTP_TRANSPORT_DEFAULT_READ_TIMEOUT
+    headers: dict[str, str] = Field(default_factory=dict)
+
+
 class MCPServerConfig(BaseModel):
     """MCP server configuration."""
 
     name: str
-    transport: Literal["sse", "stdio"]
+    transport: Literal["sse", "stdio", "streamable_http"]
     stdio: Optional[StdioTransportConfig] = None
     sse: Optional[SseTransportConfig] = None
+    streamable_http: Optional[StreamableHttpTransportConfig] = None
 
     @model_validator(mode="after")
     def stdio_or_sse_specified(self) -> Self:
```

Comment:
@onmete @asamal4 we can update the function name , now that we added streamable http